src/frontend/apps/standalone_site/src/features/Contents/features/ClassRoom/components/Manage/ClassRoomCreateForm.tsx

```diff
@@ -113,22 +114,27 @@ const ClassroomCreateForm = () => {
       <Form
         onSubmitError={() => ({})}
         onSubmit={({ value }) => createClassroom(value)}
-        onChange={(values) => {
-          setClassroom(values);
-        }}
+        onChange={(values) => setClassroom(values)}
         messages={{
           required: intl.formatMessage(messages.requiredField),
         }}
         value={classroom}
       >
-        <FormField
-          label={intl.formatMessage(messages.titleLabel)}
-          htmlFor="title-id"
-          name="title"
-          required
-        >
-          <TextInput size="1rem" name="title" id="title-id" />
-        </FormField>
+        <Field className="mb-s" fullWidth>
+          <Input
+            aria-label={intl.formatMessage(messages.titleLabel)}
+            fullWidth
+            label={intl.formatMessage(messages.titleLabel)}
+            name="title"
+            required
+            onChange={(e) => {
+              setClassroom((value) => ({
+                ...value,
+                title: e.target.value,
+              }));
+            }}
```

Comment:
Why do we need to set an onChange ? Isn't the form one triggered already ?

Reply:
No because the `Form` component is a wrapper on the grommet `Form`, the `onChange` on the `Form` is working only with grommet children components.
I could change the `Form` to a normal `form` component but we need to refacto the `Select` playlist and the `TextArea`, so I thought to keep this change for the future.

Reply:
OK, thanks for clarify this.